src/ViewInstance.js

```diff
@@ -476,7 +477,7 @@ class ViewInstance extends React.Component {
         </MenuSection>
 
         {
-          canCreateEditDeriveMARCRecord && (
+          (canCreateMARCHoldingsForInstanceWithSourceMARC || canEditDeriveMARCRecord) && (
```

Comment:
Same here for `Edit in quickMARC`

Reply:
Hmm, but this condition is not for `Edit in quickMARC` but for showing quickMARC section in the Instance record Actions menu. It needs to be displayed when user can create MARC Holdings OR when user can edit/derive MARC bib. And then each option of this menu section will or will not be displayed according to user permissions. Correct me, if I'm wrong, please :)

Reply:
Yes, you're right. I thought this was just for editing in quick marc